### PR TITLE
add conditional execute modifier

### PR DESCRIPTION
**Pull request by Steffan153**
so we can get an idea of how modifiers will work

i will add more modifiers to this PR in a bit

### CODE REVIEW

**Comment by Steffan153** [comment]
@cgccuser the problem is that the `-` element is bork. it adds instead of subtracting. and hey, look at that, the `+` operator does the same as the `-` operator:

https://github.com/Vyxal/VyxalS/blob/6a827cb3f304c36898f1836fb8f0dd85af174d8f/shared/src/main/scala/vyxal/VNum.scala#L6-L14

File: shared/src/main/scala/vyxal/ModifierFunctions.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package vyxal
+
+import vyxal.Helpers.toBool
+import vyxal.Interpreter.execute
+
+object ModifierFunctions {
```
Comment (cgccuser) [inline shared/src/main/scala/vyxal/ModifierFunctions.scala:6]: I think all of these could just be inlined in `Builtins.scala`. They look a bit odd as methods and the only place they're going to be used is in that Builtins file anyway.

**Review by cgccuser — APPROVED** [review]
I'm gonna just approve this real quick and make any changes that are needed later. Doesn't seem to be anything wrong, just a few stylistic things that we might want to change

File: shared/src/main/scala/vyxal/ModifierFunctions.scala (hunk 1 above)
Comment (Steffan153) [reply shared/src/main/scala/vyxal/ModifierFunctions.scala:6]: you mean Modifiers.scala? I thought it would just keep things organized, idk

**Pull request merged**